src/components/sequencing/form/SelectedCommand.svelte

```diff
@@ -46,12 +47,12 @@
   let missingArgDefArray: FswCommandArgument[] = [];
   let timeTagNode: TimeTagInfo = null;
 
-  $: commandNode = getAncestorStepOrRequest(node);
-  $: commandNameNode = getNameNode(commandNode);
+  $: commandNode = commandInfoMapper.getContainingCommand(node);
+  $: commandNameNode = commandInfoMapper.getNameNode(commandNode);
```

Comment:
Can we explicitly declare the type of this before using in the reactive statement?

Reply:
fixed in https://github.com/NASA-AMMOS/aerie-ui/pull/1472/commits/91d12895a8a322c90684f4a5e72170232e929b60